collector/nodes_response.go

```diff
@@ -37,7 +37,7 @@ type NodeStatsNodeResponse struct {
 	ThreadPool       map[string]NodeStatsThreadPoolPoolResponse `json:"thread_pool"`
 	JVM              NodeStatsJVMResponse                       `json:"jvm"`
 	Breakers         map[string]NodeStatsBreakersResponse       `json:"breakers"`
-	HTTP             map[string]interface{}                     `json:"http"`
+	HTTP             *NodeStatsHTTPResponse                     `json:"http"`
```

Comment:
Why is this a pointer?

Reply:
In order to change this condition:
if len(node.HTTP) == 0 {
to if node.HTTP == nil {

Reply:
Hello, @sysadmind, any update/concern on this ?